be/src/olap/utils.cpp

```diff
@@ -1424,6 +1425,24 @@ int Errno::no() {
     return errno;
 }
 
+static Status disk_error(const std::string& context, int16_t err) {
```

Comment:
We have this code in env/env_posix.cpp, should reuse it other than coping it again.